alpha/engagements/2024/Rust Foundation/update-2024-06.md

```diff
@@ -0,0 +1,52 @@
+
+### Crate repo verification
+
+After the XZ backdoor vulnerability, we shifted our security initiative focus a bit to focus more on the supply chain side. Adam is working on [provenance](https://lawngno.me/blog/2024/06/10/divine-provenance.html) tracking, verifying that a given crate is actually associated with the repository it claims to be. In addition to catching innocuous mistakes in the crate metadata, this will limit fraudulent crate creators trying to hide their malicious crates behind seemingly valid repos, when, in fact, the code for these crates are actually in unknown repos. Once caught, these crates can be quarantined and deleted using recently implemented admin functionality. 
+
+### CVE categorization addition to Painter
+
+Walter also added a RustSec CVE categorization to Painter to allow the tool to output which CVE types are the most prevalent in the Rust community and to aid future vulnerability response efforts. 
+
+### Crates.io database change
+
+The crates.io database was migrated from Heroku Postgres to Crunchy Data Postgres for both cost reduction and efficiency purposes. 
+
+### Crates.io RSS Feed
+
+An RSS feed is being added to crates.io. The [PR](https://github.com/rust-lang/crates.io/pull/8908) creates an RSS feed published at https://static.crates.io/rss/updates.xml. The feed is synced with the database in a background job after every successful publish of a new version. It includes the latest 100 published versions with the crate name, version number, crate description, URL and publish date. The feed is created via https://github.com/rust-syndication/rss.
+
+## Announcements and Community
+
+### Safety Critical Consortium
+
+The Foundation [announced](https://foundation.rust-lang.org/news/announcing-the-safety-critical-rust-consortium/) a Rust Safety Critical Consortium with nine founding members to focus on encouraging Rust in safety critical applications. Interest in the consortium is abundant and overwhelming, with over twenty more wanting to join or collaborate, and more coming every day. At RustConf, the consortium plans to define a charter and goals, along with a mechanism and process to efficiently add new members to the consortium.
+
+### Rust/C++ Interop Engineer Lead on Staff
+
+[Jon Bauman](https://foundation.rust-lang.org/news/welcoming-rust-c-interoperability-engineer-jon-bauman-to-the-rust-foundation-team/) is our new software engineer lead helping manage our all-important, Google funded [Rust/C++ interop initiative](https://foundation.rust-lang.org/news/welcoming-rust-c-interoperability-engineer-jon-bauman-to-the-rust-foundation-team/). In his first month in the role, Jon has met with more than a dozen people with a vested interest or expertise in interop, and is starting to plan a proposed interop strategy.
```

Comment:
Depending on how you want to parse "met" and whether one includes only Foundation-external stakeholders this number could be as low as 10 or as high as 18, but I doubt anybody's going to come asking for receipts.

Reply:
Thanks. Yeah, I didn't know the exact number and "dozen" felt about right.